scripts/goby_launch

```diff
@@ -157,28 +156,6 @@ if [ ! -e "$launchfile" ]; then
     echo "Error: must provide a valid launchfile: \"$launchfile\" does not exist. For help use -h."
     exit 1
 fi
-          
-# extract [goby_launch] variables
-launch_vars_regex="# *\[goby_launch\]: *"
-while IFS= read -r line
-do
-    key=$(echo $line | cut -d "=" -f 1)
-    value=$(echo $line | cut -d "=" -f 2)
-    case $key in
-        not_goby)
-            # read values into "not_goby" array: commands matching this will not use goby_terminate to quit them
-            IFS=';, ' read -r -a not_goby <<< "$value" 
-            for ((i=0; i<${#not_goby[@]}; i++))
-            do not_goby[$i]="^${not_goby[$i]}"
-            done
-            not_goby_regex=$(IFS="|"; echo "${not_goby[*]}")
-            ;;
-        *)
-            echo "Invalid [goby_launch]: directive key: " $key
-            exit 1
-            ;;
-    esac                
-done < <(egrep "$launch_vars_regex" "$launchfile" | sed "s/$launch_vars_regex//")
 
 # search for gobyd so we can use the same interprocess { } configuration for goby_terminate
 gobyd_line=$(cat "$launchfile" | egrep "^ *[/a-zA-Z0-9_]*gobyd")
```

Comment:
Not part of this commit but this part seems really fragile. Could you use a `[]` directive here too?